homeassistant/components/fritz/button.py

```diff
@@ -106,3 +127,63 @@ def __init__(
     async def async_press(self) -> None:
         """Triggers Fritz!Box service."""
         await self.entity_description.press_action(self.avm_wrapper)
+
+
+async def _async_wol_buttons_list(
+    avm_wrapper: AvmWrapper,
+    data_fritz: FritzData,
+) -> list[FritzBoxWOLButton]:
+    """Add new WOL button entities from the AVM device."""
+    _LOGGER.debug("Setting up %s buttons", BUTTON_TYPE_WOL)
+
+    new_wols: list[FritzBoxWOLButton] = []
+
+    if avm_wrapper.unique_id not in data_fritz.wol_buttons:
+        data_fritz.wol_buttons[avm_wrapper.unique_id] = set()
+
+    for mac, device in avm_wrapper.devices.items():
+        if _is_tracked(mac, data_fritz.wol_buttons.values()):  # pragma: no cover
```

Comment:
We shouldn't ignore coverage on individual lines. There's no reason to do that here. Probably test this by updating the device so that this function is called again for the same mac address.
```suggestion
        if _is_tracked(mac, data_fritz.wol_buttons.values()):
```